setup.py

```diff
@@ -1,20 +1,30 @@
 from flask import Flask
-from controllers.auth import controller_endpoints
+import os
+from controllers.auth import controller
 from flask_sqlalchemy import SQLAlchemy
 from flask_migrate import Migrate
+from sqlalchemy import VARCHAR, DateTime, text
+from flask_cors import CORS
 
 
 app = Flask(__name__)
-app.config['SQLALCHEMY_DATABASE_URI'] = 'postgresql+psycopg2://jonas:password@localhost:5432/jonas'
+CORS(app)
+app.config['SQLALCHEMY_DATABASE_URI'] = 'postgresql+psycopg2://'+os.getenv('POSTGRES_USER')+':'+os.getenv('POSTGRES_PASSWORD')+'@db:5432/jonas'
 db = SQLAlchemy(app)
 migrate = Migrate(app, db, directory='config/db/postgres/migrations')
 
-class User(db.Model):
+class Users(db.Model):
```

Comment:
Why did this change from singular to plural?

Reply:
In PostgreSQL by default exists a  reserverd word called user, this is the reason why i changed to users